src/deparse.c

```diff
@@ -0,0 +1,220 @@
+#include <postgres.h>
+#include <catalog/namespace.h>
+
+#include <fmgr.h>
+#include <utils/rel.h>
+#include <commands/dbcommands.h> //get_database_oid(dbname)
```

Comment:
Please remove the comments next to the includes. Not in our style.

Reply:
ok, would be useful though